pkg/tagger/collectors/workloadmeta_test.go

```diff
@@ -884,6 +884,31 @@ func TestHandleContainer(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "opencontainers image source",
```

Comment:
PR looks good to me, but can you please merge this test with "opencontainers image revision" for code compactness? Maintaining too many related but separate tests gets expensive in the long term.